modules/dxp/apps/segments/segments-experiment-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -23,6 +23,7 @@
 		<span class="font-weight-bold"><liferay-ui:message key="ab-test" /></span>
 
 		<clay:button
+			aria-label="Close"
```

Comment:
Same here, I think you can use LanguageUtil as [here ](https://github.com/brianchandotcom/liferay-portal/blob/master/modules/apps/commerce/commerce-address-web/src/main/resources/META-INF/resources/view.jsp#L65). Don't forget to check if close language key is defined :)